code-challenges.js

```diff
@@ -13,21 +31,66 @@ var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+const sortedOdds = (array) => {
+  //use a filter to traverse through the array
+  // find the odd numbers first
+  // finally sort low to high
+  return array.filter(value => ((typeof value === 'number') && (value % 2 !== 0))).sort((a,b) => a-b)
+}
+
+console.log(sortedOdds(fullArr1))
+console.log(sortedOdds(fullArr2))
 
 
 
 // --------------------3) Create a function that takes in a string of a single word and returns the middle letter of the word. If the word is an even number of letters, return the two middle letters.
 
+const midLetter = (string) => {
+  // local array to hold temp values
+  let tempString = []
```

Comment:
this line is actually not really necessary here. we can just declare variable `tempString` on the next line if you'd like!